prog/github/github_repository_nexus.rb

```diff
@@ -91,6 +91,18 @@ def cleanup_cache
       .limit(200)
       .destroy
 
+    # Destroy cache entries if it is created 30 minutes ago
+    # but couldn't committed yet. 30 minutes decided as during
+    # our performance tests uploading 10GB of data (which is
+    # the max size for a single cache entry) takes ~8 minutes at most.
+    # To be on the safe side, ~2x buffer is added.
+    twenty_minutes_ago = Time.now - 30 * 60
```

Comment:
It's `thirty_minutes_ago` now :D

Reply:
Since it's used only once, you can make it inline too. `seven_days_ago` is a variable because it's used multiple times.